templates/vue/src/components/modals/HelpModal/index.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <b-modal
+        id="help-modal"
+        data-qa="help-modal"
+        :visible="show"
+        size="lg"
+        title="Getting Started"
+        @hidden="$emit('close')"
+    >
```

Comment:
I suggest removing the padding around the iframe here. (see [bv docs](https://bootstrap-vue.org/docs/components/modal#variants))
```suggestion
        body-class="p-0"
    >
```